test/utils/runners.go

```diff
@@ -931,8 +931,8 @@ type CountToStrategy struct {
 }
 
 type TestNodePreparer interface {
-	PrepareNodes() error
-	CleanupNodes() error
+	PrepareNodes(nextNodeIndex int) error
+	CleanupNodes()
```

Comment:
why was the error dropped? asymmetry on prepare/cleanup is unexpected. is there another way the cleanup can fail the test if unexpected errors are encountered?

Reply:
The TestNodePreparer interface's implementations are pending a refactor (see #93794). However, there's no reason why the interface's CleanupNodes can't be left unchanged for now. Done :+1: